pom.xml

```diff
@@ -479,9 +474,6 @@
          <jettyMaxVersion>9.5</jettyMaxVersion>
       </properties>
       <!-- build against a known target platform -->
```

Comment:
Remove the comment together (also for the other two profiles).

That said, I wonder how these profiles will know and select which target platform to use.

Reply:
That's done in the the next set of line for each of those, and why the comment is still relevant.